### PR TITLE
Give more information in the error messages for failed wasm tests

### PR DESCRIPTION
**Pull request by coolreader18**

### CODE REVIEW

File: vm/src/vm.rs [hunk 1]
```diff
@@ -303,33 +303,31 @@ impl VirtualMachine {
...
-                    "__import__ not found".to_string(),
-                )),
-            }
-        }
+        let sys_modules = self.get_attribute(self.sys_module.clone(), "modules")?;
+        sys_modules.get_item(module.to_string(), self).or_else(|_| {
+            let import_func = self
+                .get_attribute(self.builtins.clone(), "__import__")
+                .map_err(|_| {
+                    self.new_exception(
```
Comment (palaviv) [inline vm/src/vm.rs]: I feel bad commenting on this as I wrote this but if you make it nicer we can use `vm.new_import_error` :smiling_imp:

File: wasm/demo/package.json [hunk 2]
```diff
@@ -24,7 +25,7 @@
         "build": "webpack",
         "dist": "webpack --mode production",
         "test": "cd ../tests; pipenv run pytest",
-        "ci": "start-server-and-test dev http-get://localhost:8080 test"
+        "ci": "start-server-and-test 'npm run build && serve dist/ -np 8080' :8080 test"
```
Comment (palaviv) [inline wasm/demo/package.json]: It seems like it does not recognize the `-np` param in travis. Maybe we need a newer version there?

Comment (coolreader18) [reply wasm/demo/package.json]: No, I just thought it supported flag concatenation without checking, it should just be `-n -p 8080`.

File: vm/src/vm.rs [hunk 3]
```diff
@@ -303,33 +303,26 @@ impl VirtualMachine {
...
-                    ],
-                ),
-                Err(_) => Err(self.new_exception(
-                    self.ctx.exceptions.import_error.clone(),
-                    "__import__ not found".to_string(),
-                )),
-            }
-        }
+        let sys_modules = self.get_attribute(self.sys_module.clone(), "modules")?;
+        sys_modules.get_item(module.to_string(), self).or_else(|_| {
```
Comment (windelbouwman) [inline vm/src/vm.rs]: Why did you change the original code to a lambda function with the `or_else` statement? Is there a performance benefit of using a lambda function? Personally I find the code a little harder to understand when a lot of lambda functions are involved.

File: vm/src/vm.rs [hunk 4]
```diff
@@ -303,33 +303,26 @@ impl VirtualMachine {
...
+            let import_func = self
```
Comment (windelbouwman) [inline vm/src/vm.rs]: First evaluating `import_func` is a good idea!

File: vm/src/vm.rs [hunk 5]
```diff
@@ -303,33 +303,26 @@ impl VirtualMachine {
...
+                .get_attribute(self.builtins.clone(), "__import__")
+                .map_err(|_| self.new_import_error("__import__ not found".to_string()))?;
+            self.invoke(
+                import_func,
+                vec![
+                    self.ctx.new_str(module.to_string()),
+                    if let Some(frame) = self.current_frame() {
```
Comment (windelbouwman) [inline vm/src/vm.rs]: Maybe fetch the current frame first, just like fetching the `import_func` first, to reduce the size of the `self.invoke` expression.

File: vm/src/vm.rs (hunk 3 above)
Comment (coolreader18) [reply vm/src/vm.rs]: I personally find it more readable because I think the intent is clearer. "Get it from `sys.modules`, or else if that fails try calling the import function.

Comment (windelbouwman) [reply vm/src/vm.rs]: Hmm, I guess it is a matter of personal preference/opinion? Maybe we could settle on having both ways, and leaving the code in either style when touching the code? Does that sound good to you? I'll do my best on this strategy at least.
